app/services/subscriptions/create_service.rb

```diff
@@ -91,7 +91,9 @@ def downgrade?
     end
 
     def should_be_billed_today?(sub)
-      sub.active? && sub.subscription_at.today? && plan.pay_in_advance? && !sub.in_trial_period?
+      return false unless sub.subscription_at.today? || sub.subscription_at.past?
```

Comment:
I might miss some context here, but in my mind the current behavior was intentional, to allow migration existing subscriptions from an external tool into Lago without billing them.

Reply:
When playing around with Lago and pay in advance plans I also noticed this and we have dedicated documentation for this behaviour as well. 

> If the plan includes a subscription fee to be paid in advance, it will be considered as already paid for the current period.

Reply:
yep - that's something in discussion - closing the PR for now